flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/copy/HiveParserSqlMinMaxAggFunction.java

```diff
@@ -39,11 +40,15 @@ public HiveParserSqlMinMaxAggFunction(
             boolean isMin) {
         super(
                 isMin ? "min" : "max",
+                null,
                 isMin ? SqlKind.MIN : SqlKind.MAX,
                 returnTypeInference,
                 operandTypeInference,
                 operandTypeChecker,
-                SqlFunctionCategory.NUMERIC);
+                SqlFunctionCategory.NUMERIC,
+                false,
+                false,
+                Optionality.FORBIDDEN);
```

Comment:
Use non-deprecated constructor which is used under the hood of deprecated one https://github.com/apache/calcite/blob/cd5229b9a371e20de207231d55bcbddf1ff39ec2/core/src/main/java/org/apache/calcite/sql/SqlAggFunction.java#L99